index.js

```diff
@@ -777,16 +789,18 @@ var tick = function() {
 						}
 					}
 				}
-				// ensure owner is correct
-				var rightfulOwner = -1;
-				for(var u in node.units) {
-					if(node.units[u] > 0) {
-						if(rightfulOwner === -1) {
-							rightfulOwner = u;
-						}
-						else {
-							// multiple owner's units, node owner unknown
-							rightfulOwner = -2;
+				if (node.owner != -3) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '!==' and instead saw '!='.](https://www.codacy.com/app/vpzomtrrfrt/turbo-spork-server/file/2651546515/issues/source?bid=3248399&fileBranchId=3307983#l792)